docs: update changelog (#779)
plentymarkets/plentyshop-pwa: docs/changelog/changelog_en.md

# Changelog plentyshopPWA

## v1.7.0 (2024-11-06) <a href="https://github.com/plentymarkets/plentyshop-pwa/compare/v1.6.0...v1.7.0" target="_blank" rel="noopener"><b>Overview of all changes</b></a>

### New

- Clicking the **Buy** button with an unsaved address now displays a notification.

> - Selecting an EU country as the shipping country now makes all EU countries available as billing options.

Reviewer: This ensures compliance with geo-blocking regulations.

> - Improved the product title display for SEO purposes. Reverts to the default if no product specific title is available.

Reviewer: This is actually a new field available on the items' data. This means you can either have a custom title or it is using the old pattern of "Item name | Company name"

- Added tax and shipping note to the wishlist page.
- Added a new request header for the `configId`.
- Added the environment variable `NO_CACHE` for disabling caching.
- The hero banner and media card now support alt text properties for images.
- Enabled font color customization for the hero banner via a template property.
- Added a new carousel compoment.
- Added an edit mode toolbar and JSON editor for the front end. Note that this is a preparatory step. Further functionality will be added in an upcoming version.

### 🩹 Fixed

- Fixed an issue where increasing the quantity beyond maximum stock would clear the cart.

> - Removed the "Add to Cart" notification from item and category pages when the quick checkout modal is not present.

Reviewer: Isn't that controlled by a setting?

Reviewer 2: @Tim-M-S 
quick checkout is not controlled by environment setting, only the timer is.

Author: According to Andrei, no.

- Corrected manufacturer data to use `externalName` instead of `name` in structured data.
- Updated headlines to use the configured font.
- Fixed layout shift on the category page.
- Resolved build script failure on Windows due to file name pattern incompatibilities.
- The build script now adds the `API_URL` to the environment if it exists.

### 👷 Changed

- Implemented a new notification design.
- Increased default notification timeout from 3 to 5 seconds.
- The `height` and `width` attributes set in Terra UI are now used only for full-size images of an item.
- Moved the generation of missing language files based on the environment's language configuration to the build script. The locale configuration is now based on the language files in the `lang` directory.
- Started unifying SDK/API error handling. Errors now return keys that can be translated in the frontend.
- Improved manufacturer visuals.
- Updated manufacturer translation text.
- Added support for a second argument to the payment status on an order. This allows you to define custom translations for different payment states.

> - If the cart is empty during the checkout process, the user is now redirected to the cart.

Reviewer: ```suggestion
- If the cart is emptied during the checkout process, the user is now redirected to the cart.
```

> - The logo container is now more flexible and adapts to the dimensions of the provided logo.

Reviewer: ```suggestion
- The logo container is now more flexible and adapts to the dimensions of the provided custom logos.
```

Author: I don't think "custom" adds any valuable information here. In the end all logos are custom logos, even the plenty one we use.

# Changelog plentyshopPWA

## v1.7.0 (2024-11-06) <a href="https://github.com/plentymarkets/plentyshop-pwa/compare/v1.6.0...v1.7.0" target="_blank" rel="noopener"><b>Overview of all changes</b></a>

### New

- Clicking the **Buy** button with an unsaved address now displays a notification.
- To ensure compliance with geo-blocking regulations, selecting an EU country as the shipping country now makes all EU countries available as billing options.
- The product title display now supports custom titles. You can set the custom title on the item. If no custom title is set, the default "Item name | Company name" is used.
- Added tax and shipping note to the wishlist page.
- Added a new request header for the `configId`.
- Added the environment variable `NO_CACHE` for disabling caching.
- The hero banner and media card now support alt text properties for images.
- Enabled font color customization for the hero banner via a template property.
- Added a new carousel compoment.
- Added an edit mode toolbar and JSON editor for the front end. Note that this is a preparatory step. Further functionality will be added in an upcoming version.

> - Added Google Pay and Apple Pay as new payment methods.

Reviewer 3: ```suggestion
- Google Pay and Apple Pay are now available as payment methods in the checkout.
```
